docker/odklite/Dockerfile

```diff
@@ -15,6 +15,9 @@ ENV PATH "/tools:/tools/dosdp-tools/bin:$PATH"
 ARG ODK_VERSION 0.0.0
 ENV ODK_VERSION $ODK_VERSION
 
+ARG ROBOT_JAR https://github.com/ontodev/robot/releases/download/v$ROBOT_VERSION/robot.jar
```

Comment:
It should be `ARG ROBOT_JAR=https://…`

While the `ENV` command accepts a form without the equal sign, the `ARG` command [does not](https://docs.docker.com/engine/reference/builder/#arg).

(Actually even for the `ENV` command the equal-less form [is discouraged](https://docs.docker.com/engine/reference/builder/#env) and may disappear in the future, so it could be a good idea at some point to update all the Docker files to make sure we always use the `=` syntax.)

Reply:
Ok, for this dockerfile i changed that! Good to know!